### PR TITLE
Use Sentry angular integrations

### LINKED ISSUE #119: Replace sentry browser package with @sentry/angular
There's a package for Angular now: https://docs.sentry.io/platforms/javascript/guides/angular/

Would be nice to also monitor performance: https://docs.sentry.io/platforms/javascript/guides/angular/#monitor-performance

Soon work will be done on the Sentry .NET SDK to continue that trace on the server.
Errors will be connected throughout the whole transaction.

### PR DESCRIPTION
**Pull request by joaopgrassi**
closes #119

### CODE REVIEW

File: src/NuGetTrends.Spa/src/app/app.module.ts [hunk 1]
```diff
@@ -13,17 +15,17 @@ import { SharedModule } from './shared/shared.module';
...
-      super.handleError(err);
-    }
-  }
-}
+Sentry.init({
+  dsn: environment.SENTRY_DSN,
+  environment: environment.name,
+  integrations: [
+    new Integrations.BrowserTracing({
+      tracingOrigins: ['localhost', 'https://nugettrends.com/', 'https://nugettrends.com/api'],
```
Comment (joaopgrassi) [inline src/NuGetTrends.Spa/src/app/app.module.ts:23]: @bruno-garcia maybe you can clarify this?  Not sure what to put in the tracing origins.. the instructions in the docs are a bit vague.

File: src/NuGetTrends.Spa/src/app/app.module.ts [hunk 2]
```diff
@@ -13,17 +15,17 @@ import { SharedModule } from './shared/shared.module';
...
+      routingInstrumentation: Sentry.routingInstrumentation,
+    }),
+  ],
+  tracesSampleRate: 1.0, // what value should be used here for prod? Docs are not clear..
```
Comment (joaopgrassi) [inline src/NuGetTrends.Spa/src/app/app.module.ts]: Also this: The docs says: 
> // Be sure to lower this in production

You know a good value to use in Production? I guess we don't want to send all traces?

Comment (bruno-garcia) [inline src/NuGetTrends.Spa/src/app/app.module.ts]:
```suggestion
  tracesSampleRate: 1.0, // Let's start with 1.0 (capture all traces) and we see how our quota goes
```

**Review by bruno-garcia — APPROVED** [review]
❤️

File: src/NuGetTrends.Spa/src/app/app.module.ts (hunk 2 above)
Comment (joaopgrassi) [reply src/NuGetTrends.Spa/src/app/app.module.ts]: Alright 👍

**Pull request merged**